albinos-cli/source/app.d

```diff
@@ -0,0 +1,86 @@
+import std.stdio;
+import std.string;
+import std.socket : UnixAddress, Socket, SocketType, AddressFamily, SocketOSException;
+import std.uni : isWhite;
+import std.process : executeShell;
+
+///
+class Client
+{
+	///
+	this(string socket_path)
+	{
+		this.address_ = new UnixAddress(socket_path);
```

Comment:
I'd rather you use the C api. I know it's not ready yet. So I would start by focusing on the parsing of the commands.

Reply:
Yeah Probably, it's here for my test on purpose to be honnest